packages/ui/src/documents/__tests__/ListDocuments.test.tsx

```diff
@@ -192,32 +203,6 @@ describe('List Documents', () => {
 
   });
 
-  it('Should edit document when clicking Edit', async () => {
```

Comment:
Same as above (test coverage deterioration)

Reply:
Edit Document should not be enabled upon the first rendering of the DocumentList